variants/picomputer-s3/platformio.ini

```diff
@@ -27,6 +27,7 @@ build_flags = ${esp32_base.build_flags}
 	-D PICOMPUTER_S3
   -D MESHTASTIC_EXCLUDE_CANNEDMESSAGES=1
   -D MESHTASTIC_EXCLUDE_INPUTBROKER=1
+  -D MESHTASTIC_EXCLUDE_BLUETOOTH=1
```

Comment:
Will there be an option on device to turn on/off the Bluetooth? 
The screen is hard to impossible to read at times and the only way to be able to do anything in that case is to connect to a phone with a better screen...
🙏

Reply:
Currently, you can connect a phone using WLAN. That should work.

Reply:
Wifi is not supported on iOS :(

Reply:
It works with Android. Another option is to use the admin channel for configuration. If you can't read the screen because of some eye related issues I think this device is not the right one for you. Instead get a LoRa device without screen (e.g. a RAK19003+RAK4631, they have huge discount at the moment) and use your phone to operate it.

Reply:
Yeah it's always what Android can do... I'm using iOS though... 
This device is great, especially when the 3 days of darkness (no electricity, not internet...) will be upon us...
All I was asking is whether you would add an option so ppl can chose to use BLE or not since I see you are completely disabling it and it was working before... 
I have a battery with my T-Deck that can get me 2 weeks with BLE on...

Reply:
There is a limitation in the firmware that let's either use BLE app or the TFT app but not both at the same time, and that is not an easy fix.